base/benchmarks/main_memory-bandwidth/metax/env.sh

```diff
@@ -0,0 +1 @@
+echo "NVIDIA PLACEHOLDER ENV.SH"
```

Comment:
please not use "NVIDIA PLACEHOLDER". if necessary, you can use "METAX PLACEHOLDER", or just simply remove the file